src/mlpack/core/tree/binary_space_tree/binary_space_tree_impl.hpp

```diff
@@ -877,7 +879,43 @@ template<typename MetricType,
          template<typename BoundMetricType, typename...> class BoundType,
          template<typename SplitBoundType, typename SplitMatType>
              class SplitType>
+size_t BinarySpaceTree<MetricType, StatisticType, MatType, BoundType,
+    SplitType>::PerformSplit(
+    MatType& data,
+    const size_t begin,
+    const size_t count,
+    const typename UBTreeSplit<BoundType<MetricType>,
+                               MatType>::SplitInfo& splitInfo)
+{
+  return SplitType<BoundType<MetricType>, MatType>::PerformSplit(data, begin,
+      count, splitInfo);
+}
```

Comment:
Sure, that sounds good to me.  I figured it might be ok in `math` since maybe somewhere else might use it someday, but I agree, maybe we shouldn't move it there until multiple places use it.